roles/wildfly_systemd/tasks/selinux_config.yml

```diff
@@ -0,0 +1,23 @@
+- name: Ensure PID directory exists
+  ansible.builtin.file:
+    path: "{{ wildfly_systemd.selinux_config.pid_path }}"
+    state: directory
+    mode: '0755'
+
+- name: Set SELinux context for the pid file
+  ansible.builtin.file:
+    path: "{{ wildfly_systemd.selinux_config.pid_path }}/{{ instance_name }}.pid"
+    owner: root
+    seuser: system_u
+    serole: object_r
+    setype: var_run_t
+    selevel: s0
+    mode: '0755'
+    state: touch
+
+- name: Store PID into the PID file
```

Comment:
I have tried without using this task but the PID is not getting stored in the pidfile.

Reply:
Mode of pidfile should be 644 or maybe 600 not 755. Anyway yeah there should be no touch, because the pidfile should only exist when the service is running. where is the pidfile created (ie. by what process?)